pkg/providers/vault/examples/pv-keyvault-csi.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: pv-keyvault
+spec:
+  capacity:
+    storage: 1Gi
+  accessModes:
+    - ReadOnlyMany
+  persistentVolumeReclaimPolicy: Retain
+  csi:
+    driver: keyvault.csi.k8s.com
+    readOnly: true
+    volumeHandle: kv
+    volumeAttributes:
+      providerName: "vault"
+      roleName: "example-role"
+      vaultAddress: "http://10.0.63.109:8200"
+      vaultSkipTLSVerify: "true"
```

Comment:
Does `vaultSkipTLSVerify: "false"` work? If so, how? If not, let's add a //TODO and an issue to track this.

Reply:
yea, that should be part of the documentation of the provider.